handbook/engineering/hiring/software-engineer-frontend.md

```diff
@@ -46,10 +55,18 @@ Learn more about what it is like to work at Sourcegraph by reading [our handbook
 ## Interview process
 
 1. You [apply here](https://jobs.lever.co/sourcegraph/b2f9a8b0-cc06-4629-81a0-0f2fa64271c7/apply).
+1. We screen your application for suitability for the role by looking at:
+   1. Your past work experience. In the absence of professional work experience we would look at any open-source projects you have created or contribute to.
+   1. Whether you have the necessary technical skills that we require.
+   1. The answers you provide to the application questions.
+   1. And the [Nice-to-haves](#nice-to-haves) we listed above.
 1. We set up a 30-minute call to learn more about what you are looking for, tell you about Sourcegraph, and answer any questions that you have.
-1. You complete a 3-hour [exercise project](software-engineer-coding-exercise.md#frontend-coding-exercise) that we designed to test the technical qualifications listed above.
-1. We review your submission and schedule a **1 hour follow-up call** to discuss it.
-1. We schedule additional interviews across multiple days.
+1. You complete a 3-hour [coding excercise](software-engineer-coding-exercise.md#frontend-coding-exercise) that we designed to test the technical qualifications and responsibilities of the role as [listed above](#your-responsibilities).
+   1. _Note: while we do not disqualify candidates who do not have React and TypeScript experience from taking the excercise, you will be required to use these for the excercise._
+1. We review your submission and schedule a **1 hour follow-up call** with an engineer and designer to discuss it.
+   1. _Note: at this point, based on the evaluation of the coding excercise assessment, we might choose to allocate you to a different team than you might have indicated a preference for during your application if we believe your skills would be more suitable in it._
+1. We set up a 1-hour interview with the **Hiring Manager** who will tell you more about the role in the team. They will ask you about your past work and accomplishments and assess your sutiability for the role.
```

Comment:
@withdavidli this is a new step in the interview process for the Frontend Engineer role.

@nicksnyder It's my recommendation that if a candidate does not receive a strong hire from either the hiring manager or the engineer & designer that reviewed their coding exercise that we do not proceed with the rest of the interviews. This will ensure we have a high bar for joining the team and protect the time of other teammates so they only interview candidates that are very likely to result in a positive outcome.

Reply:
> It's my recommendation that if a candidate does not receive a strong hire from either the hiring manager or the engineer & designer that reviewed their coding exercise that we do not proceed with the rest of the interviews. This will ensure we have a high bar for joining the team and protect the time of other teammates so they only interview candidates that are very likely to result in a positive outcome.


100% agree with this sentiment, but I have questions on implementation. For example, I don't think it is a good outcome to simply add another 1hr interview to our process.

Example alternatives that I would be more comfortable with:
1. Don't add this step, but instead make the first HM call this call (make it 1 hour and do a heavy screen). You can always cut the call short if it isn't going well. We are in a strong hiring position, we have a public handbook, so our first call with a candidate doesn't have to be pitching the candidate as much, we can go in and immediately start asking tough questions. Set candidate expectations in the email template when we reach out to them to schedule this call (e.g. we expect you have already read the handbook).
1. If we add a second HM step, it needs to be a structured interview just like other interviews (documented questions and expectations). To a first approximation it could be as simple as "HM does technical experience interview" so this would be net-neutral change to candidates. In any case, if the HM is going to do another 1 hr interview, we need to ensure it doesn't duplicate signal from subsequent interviews.

Reply:
@nicksnyder I'm confused why you say "second HM step". This is the first and only time the HM gets to meet the candidate in the process afaict?

Reply:
Oh, it wasn't clear from the JD that a recruiter does the first 30m call (step 3). If that is true, then it should be explicit.

> We set up a 30-minute call to learn more about what you are looking for, tell you about Sourcegraph, and answer any questions that you have.

If this is a recruiter call, then I think the HM should speak with the candidate BEFORE the coding exercise.

Reply:
I've clarified the roles and moved the HM call to be before the coding exercise.